libchisel/src/checkstartfunc.rs

```diff
@@ -14,6 +14,25 @@ impl CheckStartFunc {
     }
 }
 
+use std::collections::HashMap;
+
+impl ModuleConfig for CheckStartFunc {
+    fn with_defaults() -> Result<Self, ModuleError> {
```

Comment:
We could just implement this via `Default`, I think.

Reply:
I know, but the point is to require it for every module to avoid the need for more magic in `ChiselModule`.

Reply:
And this one has a `Result`.

Reply:
Why would the default be able to fail?

Reply:
nvm